### ISSUE TITLE
Hide pictures from the member picture page 

### ISSUE THREAD

**Issue by BHenkemans** [GEWIS/gewisweb#1887]
### What would you like?

I want the ability to hide pictures from the member picture page. This allows a member to hide pictures (they do not like) from their profile page. This would not hide the picture from the original album. The picture would be grayed out on your own personal page.

As an added bonus, we could also configure that if a picture is part of this list, that it cannot become the picture of the week.

### Why is this needed?

It gives a member some control over what pictures their profile shows when they are looked up by other members via the Find a member feature

**Comment by tomudding** [comment]
Closely related to GH-1410. Potentially part of GH-148.

However, I do want to note that _"As an added bonus, we could also configure that if a picture is part of this list, that it cannot become the picture of the week."_ is likely infeasible due to the fact there is no way we can ensure that the person tagged is actually in the picture and it is prone to abuse (there are plenty of pictures in which people are tagged that are not actually present in the picture).

**Comment by tomudding** [comment]
For future reference, the flag has to be on the photo and not the tag (to prevent people from re-applying the tag if they notice a specific photo is hidden).

**Comment by BHenkemans** [comment]
GH-1410 is slightly different but can be implemented while implementing this feature (as part of a privacy feature pack). This issue reads as wholly opting out of tagging. This issue could be a middle ground. Allowing members to hide tags could remove one of the objections to being tagged, which is that their member picture page is filled with undesirable pictures.

I do agree with your abuse comment. Perhaps that it is best to implement GH-1806 instead.

**Comment by tomudding** [comment]
> Allowing members to hide tags could remove one of the objections to being tagged, which is that their member picture page is filled with undesirable pictures.

I agree, and for _more_ future reference, the photos should probably removed from the profile page and not blurred. This removes complexity from having to deal with the photo viewer and will ensure that people are less likely to go and search for which photo is "hidden".

**Comment by tomudding** [comment]
Last _for future reference_ (hopefully): when viewing your own profile, the images are shown but with an overlay (`fa-eye-slash`) if they are hidden.
